drivers/bridge/bridge.go

```diff
@@ -466,9 +474,7 @@ func parseNetworkGenericOptions(data interface{}) (*networkConfiguration, error)
 		config = opt
 	case map[string]interface{}:
 		config = &networkConfiguration{
-			EnableICC:          true,
-			EnableIPTables:     true,
-			EnableIPMasquerade: true,
```

Comment:
Yes as discussed, we have to keep EnableIPMasquerade as part of the network config.